### DISCUSSION TITLE
购买2年版账号后无法正常登录

### DISCUSSION THREAD

**Discussion by rionadmin** [getlantern/lantern-forum-cn#726] [General]
购买两年专业版后，用注册邮箱登录始终不发验证码以至于无法登录到专业版账号，交易信息如下图，现申请原价退款，请原路退还。
![微信图片_20240615004650](https://github.com/user-attachments/assets/bd36b0b2-7c86-43a7-a422-5cf8671dd612)

**Comment by MaxMKMax** [comment]
@rionadmin 您好. 非常抱歉影响了您的体验感! 请您息怒, 我帮您解决问题. 先别退款, 给我们个机会好吗? 你提供我您的账号邮箱, 我帮您拿验证码.谢谢!

**Reply by MaxMKMax** [reply]
@rionadmin 
您好. 这是您的激活码:
DJFHY-76MWV-XTTDQ-8TJ44-6WHQJ

**Reply by kidokogills** [reply]
为什么他有激活码，我的一直不给发？

**Comment by rionadmin** [comment]
我在点击注册专业版用户时出现“兑换激活码时错误”提示，什么问题，如何解决

Mike K ***@***.***> 于2024年7月26日周五 15:21写道：

**Comment by MaxMKMax** [comment]
@rionadmin 请您提供我您的账号邮箱. 我们重新发激活码给您.

**Comment by rionadmin** [comment]
我的账号邮箱是：

   - ***@***.***


   -



Mike K ***@***.***> 于2024年8月6日周二 02:51写道：

**Comment by rionadmin** [comment]
处理时间过慢，再不提供激活码的话请退款

Mike K ***@***.***> 于2024年8月6日周二 02:51写道：
